doc/source/serve/autoscaling.md

```diff
@@ -0,0 +1,43 @@
+(serve-autoscaling)=
+
+# Serve Autoscaling
+
+This section should help you:
+
+- Understand how Ray Serve autoscaling works
+- Understand config parameter usage
+
+
+## Autoscaling architecture
+Ray Serve autoscaling is to increase and decrease the number of replicas based on the load, and ray serve provide a couple of config parameters to meet the different workload use cases.
+
+![pic](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling.svg)
+
+- Deployment handle(Client) and worker replica periodically push the stats to the autoscaler.
```

Comment:
ServeHandle (link) and deployment replicas (link to managing deployments num_replicas for now, architecture later) periodically push their queuing metrics to the Serve autoscaler.

Reply:
We need to make sure our terminology is consistent with the rest of the documentation. Users won't understand what "Deployment handle(Client)" means. Would suggest something like "Each replica with a handle to another deployment periodically pushes stats to the controller."